.github/workflows/docker-publish.yml

```diff
@@ -1,82 +1,89 @@
-# Source: https://raw.githubusercontent.com/foundry-rs/foundry/master/.github/workflows/docker-publish.yml
-name: docker
+#
```

Comment:
Some top-level documentation would be nice here, e.g. how/where to run the workflow and linking to https://github.com/argumentcomputer/zk-light-clients/actions/workflows/docker-publish.yml